DatabaseInitialScript.sql

```diff
@@ -187,6 +187,21 @@ IF NOT EXISTS (SELECT NULL FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'C
 	  ADD [Level] TINYINT NOT NULL
   GO
 
+IF NOT EXISTS (SELECT NULL FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'Course' AND COLUMN_NAME = 'ScreenCast')
+  ALTER TABLE dbo.Course
+    ADD [ScreenCast] NVARCHAR(1000) NOT NULL DEFAULT ('https://res.cloudinary.com/funfood/image/upload/v1565220505/courses/default_cover.gif')
```

Comment:
Don't default a value like this. Should use logic to set the value after reading from the db, before showing to the UI.